### PR TITLE
Homework02

### PR DESCRIPTION
**Pull request by Maziluiulia**
homework02

### CODE REVIEW

**Comment by Maziluiulia** [comment]
Homework 02.
Accidently added homework 01 also

**Review by klymenok — CHANGES_REQUESTED** [review]
Please, remove files that is not related to the homework.

Also, it would be better in your solution to check first if value is digit since words or phrases can contain some symbols (`', -`), and those symbols won't be recognized as letters during isalpha() check and you will get an error during the `int(a)` operation.

File: home02.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+a = input("Write something: ")
+if (a.isdigit()) and (int(a) % 2 == 0):
```
Comment (klymenok) [inline home02.py]: There is no need to use parentheses around the condition. You can use it like this: `if a.isdigit()) and (int(a) % 2 == 0:`

File: home02.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    print("It is number and it is even")
+elif (a.isdigit()) and (int(a) % 2 != 0):
+    print("It is number and it is odd")
+else:
+    print("It is text ")
```
Comment (klymenok) [inline home02.py]: It would be much better to format this text in one line, like this: `print(f'It is a text. Its length is {len(a)}')`

**Review by klymenok — APPROVED** [review]
Looks good. Check my comment. You can improve your code before you merge the PR, if you want

**Pull request merged**